package.json

```diff
@@ -42,11 +42,11 @@
     "bluebird"
   ],
   "dependencies": {
-    "p-timeout": "^1.1.1"
+    "p-timeout": "^2.0.1"
   },
   "devDependencies": {
     "ava": "*",
     "delay": "^1.3.1",
-    "xo": "*"
+    "xo": "^0.19.0"
   }
```

Comment:
Leave it a `*`, it's intended

Reply:
Switched back.  Honestly no idea why I touched this.